packages/base/lua/CellularSpace.lua

```diff
@@ -892,8 +1000,15 @@ CellularSpace_ = {
 
 		defaultTableValue(data, "name", "1")
 		defaultTableValue(data, "check", true)
-
-		self.cObj_:loadNeighborhood(data.source, data.name, data.check)
+        
+		if data.source:endswith(".gal") then
+			loadNeighborhoodGAL(self, data)
+		elseif data.source:endswith(".gwt") then
+			loadNeighborhoodGWT(self, data)
+		elseif data.source:endswith(".gpm") then
+			loadNeighborhoodGPM(self, data)        
+		end
```

Comment:
add an else here with an error message and remove the if between `mandatoryTableArgument` and `defaultTableValue` in the beginning of this function.